projects/ngrx.io/content/guide/migration/v8.md

```diff
@@ -0,0 +1,121 @@
+# V8 Update Guide
+
+## Angular CLI update
+
+NgRx supports using the Angular CLI `ng update` command to update your dependencies.
+
+To update your packages to the latest released version, run the command below.
+
+```sh
+ng update @ngrx/store
+```
```

Comment:
We might need a section about the migrations that are run when you update.

Reply:
Yep, I'll verify them when we've published the beta version.